libethereum/State.h

```diff
@@ -96,60 +122,62 @@ class Executive;
-		/// Change::value the storage value.
-		Storage,
-
-		/// Account storage root was modified.  Change::value contains the old
-		/// account storage root.
-		StorageRoot,
-
-		/// Account nonce was changed.
-		Nonce,
-
-		/// Account was created (it was not existing before).
-		Create,
-
-		/// New code was added to an account (by "create" message execution).
-		Code,
-
-		/// Account was touched for the first time.
-		Touch
-	};
-
-	Kind kind;        ///< The kind of the change.
-	Address address;  ///< Changed account address.
-	u256 value;       ///< Change value, e.g. balance, storage and nonce.
-	u256 key;         ///< Storage key. Last because used only in one case.
-	bytes oldCode;    ///< Code overwritten by CREATE, empty except in case of address collision.
-
-	/// Helper constructor to make change log update more readable.
-	Change(Kind _kind, Address const& _addr, u256 const& _value = 0):
-			kind(_kind), address(_addr), value(_value)
-	{
-		assert(_kind != Code); // For this the special constructor needs to be used.
-	}
-
-	/// Helper constructor especially for storage change log.
-	Change(Address const& _addr, u256 const& _key, u256 const& _value):
-			kind(Storage), address(_addr), value(_value), key(_key)
-	{}
-
-	/// Helper constructor for nonce change log.
-	Change(Address const& _addr, u256 const& _value):
-			kind(Nonce), address(_addr), value(_value)
-	{}
-
-	/// Helper constructor especially for new code change log.
-	Change(Address const& _addr, bytes const& _oldCode):
-			kind(Code), address(_addr), oldCode(_oldCode)
-	{}
+    enum Kind : int
+    {
+        /// Account balance changed. Change::value contains the amount the
+        /// balance was increased by.
+        Balance,
+
+        /// Account storage was modified. Change::key contains the storage key,
+        /// Change::value the storage value.
+        Storage,
+
+        /// Account storage root was modified.  Change::value contains the old
+        /// account storage root.
+        StorageRoot,
+
+        /// Account nonce was changed.
+        Nonce,
+
+        /// Account was created (it was not existing before).
+        Create,
+
+        /// New code was added to an account (by "create" message execution).
+        Code,
+
+        /// Account was touched for the first time.
+        Touch
+    };
+
+    Kind kind;        ///< The kind of the change.
+    Address address;  ///< Changed account address.
+    u256 value;       ///< Change value, e.g. balance, storage and nonce.
+    u256 key;         ///< Storage key. Last because used only in one case.
+    bytes oldCode;    ///< Code overwritten by CREATE, empty except in case of
+                      ///< address collision.
+
+    /// Helper constructor to make change log update more readable.
+    Change(Kind _kind, Address const& _addr, u256 const& _value = 0)
+      : kind(_kind), address(_addr), value(_value)
+    {
+        assert(_kind !=
```

Comment:
I cannot make it to break the comment instead of the expression...

Reply:
Hm, is there a way to enforce comments being on separate lines only?

Reply:
I'm guessing you can set it to the `ColumnLimit` (in this case 80 IIRC) so the formatter never appends it to the same line.